src/firmware_new/tools/communication/transfer_media/BUILD

```diff
@@ -14,5 +14,7 @@ cc_library(
         ":transfer_medium",
         "//software/multithreading:thread_safe_buffer",
         "@g3log",
+        "@boost//:array",
+	"@boost//:asio",
```

Comment:
nit: indenting not aligned (should be fixed by running formatting)

Reply:
Ran formatting, should be fixed